app/controllers/assessor_interface/assessment_sections_controller.rb

```diff
@@ -29,11 +31,22 @@ def load_assessment_section_and_assessment
             }
           )
           .find_by!(key: params[:key])
+    end
+
+    def assessment
+      @assessment ||= @assessment_section.assessment
+    end
+
+    def application_form
+      @application_form ||= assessment.application_form
+    end
+
+    def qualifications
+      @qualifications ||= application_form.qualifications.ordered
+    end
```

Comment:
Will these be accessible to the view if they're written as methods like this?

Reply:
They will not. I've missed a bit out 😮‍💨 I think your view object suggestion is probably the best way to go but I'll sort this out for now.

Reply:
I've added a view object.